tailcfg/derpmap.go

```diff
@@ -130,10 +130,15 @@ type DERPNode struct {
 	// server.
 	STUNOnly bool `json:",omitempty"`
 
-	// DERPTestPort is used in tests to override the port, instead
-	// of using the default port of 443. If non-zero, TLS
-	// verification is skipped.
-	DERPTestPort int `json:",omitempty"`
+	// DERPPort optionally provides an alternate TLS port number
+	// for the DERP HTTPS server.
+	//
+	// If zero, 443 is used.
+	DERPPort int `json:",omitempty"`
+
+	// InsecureForTests is used by unit tests to disable TLS verification.
+	// It should not be set by users.
+	InsecureForTests bool `json:",omitempty"`
```

Comment:
maybe I'm misunderstanding something, but why does this need to be exported at all if it shouldn't be set by users? Would it be more convenient to have it unexported and have a getter?

Reply:
We want to set it in tests easily.

It's only recently that users could use this. We can enforce it in policy.Parse that users can't set it.